### ISSUE TITLE
Include character meaning within Single Character Layout.

### ISSUE THREAD

**Issue by ThomasLeigh** [SYtor/Kanji-Dojo#80]
I find Single Character Layout a great improvement which finally allows me to select exactly those characters I want to practice at the time. I've observed that many times I miss the character's meaning being included within that list. 

Currently there are four lines of text next to each character on the Single Character List. Would it be all right for You to include one more line (I think above all the rest four) which could display the character meaning (it could be limited only for a single word, still it would be helpful)?

**Comment by syt0r** [comment]
I could add it but it might spoil reviews so I'm not so sure about it. The purpose of that screen is just to select characters for practice and not to review or recall their meanings after all

**Comment by ThomasLeigh** [comment]
Good point. Would it be all right to add it as an option within the app's settings, disabled by default? (e.g. "Show character meanings on the Single Character List")
